CMakeLists.txt

```
@@ -22,7 +22,7 @@ project(k4ActsTracking LANGUAGES CXX)
 
 SET( ${PROJECT_NAME}_VERSION_MAJOR 0 )
 SET( ${PROJECT_NAME}_VERSION_MINOR 1 )
-SET( ${PROJECT_NAME}_VERSION_PATCH 0 )
+SET( ${PROJECT_NAME}_VERSION_PATCH 1 )
```

```suggestion
SET( ${PROJECT_NAME}_VERSION_PATCH 99 )
```
Our usual patch version to signal that we are in an untagged development build